lib/ae_mdw/aexn_contracts.ex

```diff
@@ -208,6 +210,17 @@ defmodule AeMdw.AexnContracts do
     end
   end
 
+  defp valid_aex141_extension?("mintable_limit", functions) do
```

Comment:
to be updated along with new ref implementation also for mintable_template_limit.